samples/Scripts/AppWRLCPP_PreBuild.ps1

```diff
@@ -24,10 +25,43 @@ $OutDir = "$ProjectDir" + "inc"
 
 if(!(Test-Path "$OutDir"))
 {
+    echo "Creating directory: $OutDir"
     mkdir $OutDir 
 }
+
+$OutDir = "$ProjectDir" + "inc"
+
+$DestinationPath = ""
+if($GeneratedFilesDir -ne "")
+{
+    echo "Generated Files dir: $GeneratedFilesDir"
+    $GeneratedFilesDirFullPath = "$ProjectDir" + "$GeneratedFilesDir"
+    echo "GeneratedFilesDirFullPath: $GeneratedFilesDirFullPath"
+    if(!(Test-Path "$GeneratedFilesDirFullPath"))
+    {
+        echo "making dir $GeneratedFilesDirFullPath"
+        mkdir $GeneratedFilesDirFullPath
+	    
+    }
+    else
+    {
+		echo "removing content from $GeneratedFilesDirFullPath"
+        Get-ChildItem -Path '$GeneratedFilesDirFullPath' -Recurse | Remove-Item -force -recurse
+    }
+	
+	$DestinationPath = $GeneratedFilesDirFullPath + "winrt\"
```

Comment:
>winrt [](start = 50, length = 5)

These are not winrt headers.. maybe we call it "wrl\" or just "inc"

Reply:
for the script only, headers are generated by midl in that folder

Reply:
these headers are not winrt though.

---
In reply to: [316925564](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/53#discussion_r316925564) [](ancestors = 316925564)